core/object/make_virtuals.py

```diff
@@ -2,7 +2,7 @@
 #define GDVIRTUAL$VER($RET m_name $ARG) \\
 StringName _gdvirtual_##m_name##_sn = #m_name;\\
 mutable bool _gdvirtual_##m_name##_initialized = false;\\
-mutable GDExtensionClassCallVirtual _gdvirtual_##m_name = nullptr;\\
+mutable void* _gdvirtual_##m_name##_data = nullptr;\\
```

Comment:
Personally, I think I'd still prefer if this kept the old variable name. Since this could be either a function pointer or user data, I don't think adding `_data` to the variable name really helps with clarity - if anything it's slightly more confusing since this might _not_ be data.

Reply:
I'll change it back. I was actually thinking the opposite, that using `_data` would make it more obvious that it was doing multiple things (serving as a function pointer or serving as callback data). But I'm not super attached to a name change.